workflows/tests/aligner/hisat2.nf.test

```diff
@@ -0,0 +1,48 @@
+nextflow_pipeline {
+
+    name "HISAT2"
+    script "../../../main.nf"
+
+    test("Should run with defaults") {
+
+        tag "HISAT2"
+
+        when {
+            params {
+                outdir = "$outputDir"
+                aligner = "hisat2"
+                hisat2_index = 'https://raw.githubusercontent.com/nf-core/test-datasets/nascent/reference/GRCh38_chr21_hisat2.tar.gz'
```

Comment:
Why in this case the index is provided?

Reply:
Because I chose not to generate index in the pipeline since it'll be provided by nf-core/references, hopefully soon ™️